programs/clearing_house/src/controller/pnl.rs

```diff
@@ -112,6 +113,8 @@ pub fn settle_pnl(
     let user_unsettled_pnl: i128 =
         user.perp_positions[position_index].get_claimable_pnl(oracle_price, max_pnl_pool_excess)?;
 
+    msg!("user_unsettled_pnl {}", user_unsettled_pnl);
+    validate_spot_balances(spot_market)?;
```

Comment:
do we want `validate_spot_market_vault_amount` too?

Reply:
this was left on accident, going to delete